ethtool/collector.go

```diff
@@ -1,6 +1,6 @@
 /*
 http://www.apache.org/licenses/LICENSE-2.0.txt
-Copyright 2015 Intel Corporation
+Copyright 2016 Intel Corporation
```

Comment:
Please, do not change the date in that way, leave `2015` as the date of creation the file was created.